br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -446,6 +449,13 @@ func setDatumByString(d *types.Datum, v string, meta *parquet.SchemaElement) {
 	if meta.LogicalType != nil && meta.LogicalType.DECIMAL != nil {
 		v = binaryToDecimalStr([]byte(v), int(meta.LogicalType.DECIMAL.Scale))
 	}
+	if meta.Type != nil && *meta.Type == parquet.Type_INT96 && len([]byte(v)) == 12 {
```

Comment:
```suggestion
	if meta.Type != nil && *meta.Type == parquet.Type_INT96 && len(v) == 12 {
```

Or maybe use 96/8 to replace 12 😂

Reply:
what's the value of LogicalType if enter this branch

Reply:
No logical type is set (nil).

Reply:
2536ce5